packages/next/cra-compat.js

```diff
@@ -0,0 +1,79 @@
+module.exports = function craCompat(config) {
+  return {
+    ...config,
+
+    webpack(webpackCfg, ctx) {
+      const { webpack } = ctx
+
+      // CRA prevents loading all locales by default
+      // https://github.com/facebook/create-react-app/blob/fddce8a9e21bf68f37054586deb0c8636a45f50b/packages/react-scripts/config/webpack.config.js#L721
+      webpackCfg.plugins.push(
+        new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/)
+      )
+
+      // CRA allows importing non-webpack handled files with file-loader
+      // these need to be the last rule to prevent catching other items
+      // https://github.com/facebook/create-react-app/blob/fddce8a9e21bf68f37054586deb0c8636a45f50b/packages/react-scripts/config/webpack.config.js#L594
+
+      const isWebpack5 = config.future && config.future.webpack5
```

Comment:
This can't be used to detect webpack 5. Should be:

```suggestion
      const isWebpack5 = parseInt(ctx.webpack.version) === 5
```

Reply:
When using the CRA migration it would be a good opportunity to force webpack 5 usage as this is for a "new" app. That also means we don't get a divergence of config here.

Reply:
We can't force it when `@svgr/webpack` is used for the `import {ReactComponent as Something} from './some.svg'` since it's not yet compatible with webpack 5, it's enabled by default when this import isn't used though

Reply:
Can we codemod / ask users to change that import instead? I don't want converted apps to run on webpack 4 in any case.

Reply:
Updated in 4feec9b035690ceacb517a2ac7cd1cb48b589b5f to show an error when we detect this type of import and asks them to migrate to a normal SVG import instead to continue to prevent de-opting webpack 5